packages/reactor-api/src/subgraphs/base/index.ts

```diff
@@ -0,0 +1,31 @@
+import { ISubgraph } from "../types";
+import { DocumentNode } from "graphql";
+import { IDocumentDriveServer } from "document-drive";
+import { SubgraphArgs } from "../types";
+import { GraphQLResolverMap } from "@apollo/subgraph/dist/schema-helper";
```

Comment:
Let's reexport GraphQLResolverMap from "types.ts" to have a simpler import instead of that @apollo dist path.